ratis-client/src/main/java/org/apache/ratis/client/impl/AsyncImpl.java

```diff
@@ -51,6 +51,11 @@ public CompletableFuture<RaftClientReply> sendReadOnly(Message message, RaftPeer
     return send(RaftClientRequest.readRequestType(), message, server);
   }
 
+  @Override
+  public CompletableFuture<RaftClientReply> sendReadAfterWrite(Message message) {
+    return send(RaftClientRequest.readAfterWriteConsistentRequestType(), message, null);
```

Comment:
@SzyWilliam , Should `sendReadAfterWrite` use `UnorderedAsync.send(..)`?  Indeed, all the linearizable reads should use `UnorderedAsync`.

Reply:
That's true! Thanks for pointing out, I'll change all the linearizable reads to use `UnorderedAsync`